catalog/imaging/catalog-imaging-service-impl/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- 
+/**
+ * Copyright (c) Connexta, LLC
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>alliance-imaging</artifactId>
+        <groupId>com.connexta.alliance.imaging</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>alliance-imaging-service-impl</artifactId>
+    <packaging>bundle</packaging>
+    <name>Alliance :: Imaging :: Service :: Impl</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.connexta.alliance.imaging</groupId>
+            <artifactId>alliance-imaging-service-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            alliance-imaging-service-api
+                        </Embed-Dependency>
+                        <Export-Package/>
+                        <Import-Package>
```

Comment:
You should be able to leave the `<Import-Package>` empty once you remove the embed.